lib/message.rb

```diff
@@ -0,0 +1,25 @@
+require 'twilio-ruby'
+
+class Message
+
+  ACCT_SID = "AC175ffc1e7a9117a7fce465205f64f4e9"
+  AUTH_TOKEN = "f60ac2e885546d154b7d50867b3c8f4d"
+
+  attr_reader :to, :from
+
+  def initialize(to = '+447540139449', from = '+441202286041')
+    @to = to
+    @from = from
+    @client = Twilio::REST::Client.new ACCT_SID, AUTH_TOKEN
+  end
+
+  def send_message
```

Comment:
You can use the dependency injection technique with other people's code as well as your own.  Imagine one of your classes, A, depends on another of your classes, B.  Imagine you're testing A and to do that you need to stub B.  You can write A so that it gets B injected.  This will let you stub B.  You can use the same technique when B is not one of your classes (B) but someone else's class (Twilio).  Does that make sense?